src/model.py

```diff
@@ -331,9 +333,83 @@ def to_misp(self) -> List[MISPObject]:
 
         # filter out objects without any attributes
         to_return = list(filter(lambda x: bool(x.attributes), to_return))
-
         return to_return
 
+    def to_maco(self) -> model.ExtractorModel:
+        output = model.ExtractorModel(family=self.family)
+
+        for key in self.rsa_keys:
+            obj = model.ExtractorModel.Encryption(
+                algorithm="rsa",
+                public_key=str((key.n, key.e)),
+            )
+            if key.d:
+                obj.key = str((key.n, key.d))
+            output.encryption.append(obj)
+
+        for curve in self.ecdsa_curves:
+            output.encryption.append(
+                model.ExtractorModel.Encryption(
+                    algorithm=curve.t,  # for example, "ecdsa_pub_p384"
```

Comment:
Is there a convention for naming this types (we don't have an enum ourselves, though we map `ecdsa_pub_p384` to something that MISP wants)